docs/src/pages/blog/everything.md

```diff
@@ -0,0 +1,163 @@
+
+We need to know about the parameters of the `$everything` operation. 
+
+| Name | Type | Description |
+|---|---|---|
+| `start` | date | The date range relates to care dates, not record currency dates - e.g. all records relating to care provided in a certain date range. If no start date is provided, all records prior to the end date are in scope. |
+| `end` | date | The date range relates to care dates, not record currency dates - e.g. all records relating to care provided in a certain date range. If no end date is provided, all records subsequent to the start date are in scope. |
+| `_since` | date | Resources updated after this period will be included in the response. The intent of this parameter is to allow a client to request only records that have changed since the last request, based on either the return header time, or or (for asynchronous use), the transaction time|
+| `_type` | code | One or more parameters, each containing one or more comma-delimited FHIR resource types to include in the return resources. In the absence of any specified types, the server returns all resource types | 
+| `return` | Bundle | The bundle is "searchset" |
+
+Note, currently the implementation ignores the `_count` parameter.
+
+Now, let's prepopulate some sample data on the server using the [Integration Test data](https://github.com/IBM/FHIR/blob/main/fhir-server-test/src/test/resources/testdata/everything-operation/Antonia30_Acosta403.json).
+
+1. Download the Sample Data
+
+``` sh
+curl -L https://raw.githubusercontent.com/IBM/FHIR/main/fhir-server-test/src/test/resources/testdata/everything-operation/Antonia30_Acosta403.json -o Antonia30_Acosta403.json
+```
+
+2. Load the Sample Data bundle to the IBM FHIR Server
+
+``` sh
+curl -k --location --request POST 'https://localhost:9443/fhir-server/api/v4' \
+--header 'Content-Type: application/fhir+json' \
+--header "Authorization: Basic ${DUMMY_PASSWORD}" \
+--data-binary  "@Antonia30_Acosta403.json" -o response.json
+```
+
+3. Scan the response.json for any status that is not `"status": "201"`.  For example, the status is in the family of User Request Error or Server Side Error.
+
+4. Check the `response.json` and find the id for Patient (it should be the first one).
+
+``` json
+        {
+            "response": {
+                "id": "178f4a3f869-6abfe1e6-3a4a-4c9a-81a8-fb8c6263e003",
+                "status": "201",
+                "location": "Patient/178f4a3f869-6abfe1e6-3a4a-4c9a-81a8-fb8c6263e003/_history/1",
+                "etag": "W/\"1\"",
+                "lastModified": "2021-04-21T13:34:50.207684Z"
+            }
+        },
+```
+
+5. Check the `Patient/[id]/$everything`
+
+* Request *
+``` sh
+curl -k --location --request GET 'https://localhost:9443/fhir-server/api/v4/Patient/178f4a3f869-6abfe1e6-3a4a-4c9a-81a8-fb8c6263e003/$everything' \
+    --header 'Content-Type: application/fhir+json' \
+    --header "Authorization: Basic ${DUMMY_PASSWORD}"
```

Comment:
should we steer them toward the `--user` (or `-u`) option instead?  its a little easier than getting the base64 just right

Reply:
Fixed